src/makeCalculator.js

```diff
@@ -38,6 +38,45 @@
  */
 function makeCalculator() {
   // write code here
+  const obj = {
+    result: 0,
+
+    add(a) {
+      this.result += a;
```

Comment:
All methods should return the object itself. You should add 'return this;' at the end of the method.